python/ccxt/bybit.py

```diff
@@ -1374,84 +1341,96 @@ def fetch_order(self, id, symbol=None, params={}):
             # conditional orders ---------------------------------------------
             # 'stop_order_id': id,  # one of stop_order_id or order_link_id is required for conditional orders
         }
-        method = 'privateLinearGetOrderSearch' if self.is_linear(symbol) else 'privateGetOrder'
-        stopOrderId = self.safe_string(params, 'stop_order_id')
-        if stopOrderId is None:
-            orderLinkId = self.safe_string(params, 'order_link_id')
-            if orderLinkId is None:
+        order_type = self.safe_string_lower(params, 'type')
+        params = self.omit(params, ["type"])
+        is_conditional = order_type == 'stop'
```

Comment:
making sure, so as we said, we are not sending(and still won't) in params type == 'stop', right?